modules/aws-console-changing-aws-instance-type.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+:_content-type: PROCEDURE
+[id="aws-console-changing-aws-instance-type_{context}"]
+= Changing the Amazon Web Services instance type by using the AWS console
+
+You can change the Amazon Web Services (AWS) instance type that your control plane machines use by updating the instance type in the AWS console.
+
+.Prerequisites
+
+* You have access to the AWS console with the permissions required to modify the EC2 Instance for your cluster.
+
+.Procedure
+
+. Open the AWS console and fetch the instances for the control plane machines.
+
+. Choose one control plane machine instance.
+
+.. For the selected control plane machine, back up the etcd data by creating an etcd snapshot. For more information, see "Backing up etcd".
+
+.. In the AWS console, stop the control plane machine instance.
+
+.. Select the stopped instance, and click *Actions* -> *Instance Settings* -> *Change instance type*.
+
+.. Change the instance to a larger type, ensuring that the type is the same base as the previous selection, and apply changes. For example, you can change `m6i.xlarge` to `m6i.2xlarge` or `m6i.4xlarge`.
+
```

Comment:
I think this section also needs a notice about updating the respective Machine object in the OCP cluster if there is any. Otherwise, there will be a drift between a Machine spec and the actual cluster state.

WDYT @jeana-redhat @JoelSpeed ?

Reply:
ACK - do they need to do that before they start the instance in the next step?

@sheriff-rh I will need to fix this in `main` and CP to 4.12+, so I'll get a PR up with a note that you can stick in here too :)

Reply:
It can be done at any point in the cycle, probably after, as IIRC, that's where it is in other docs we have on this/similar topics

Reply:
Ok, added after "Start the instance." @lobziik and/or @JoelSpeed PTAL: #55485

If it's good, Andrew can just add the same in here.

Reply:
![image](https://user-images.githubusercontent.com/46728411/216422088-83553632-727d-4079-bbd2-e6edf1b8cd22.png)
How does this look @lobziik ?

Reply:
Direct link to the build preview: [Changing the Amazon Web Services instance type by using the AWS console](http://file.rdu.redhat.com/antaylor/OCPBUGS-4561/scalability_and_performance/recommended-host-practices.html#aws-console-changing-aws-instance-type_recommended-host-practices)